tests/foreman/ui/test_host.py

```diff
@@ -1568,13 +1568,13 @@ def test_positive_bulk_delete_host(session, module_loc):
             operatingsystem=host_template.operatingsystem,
             ptable=host_template.ptable,
         ).create().name
-        for _ in range(18)
+        for _ in range(3)
```

Comment:
Why we are changing from 18 to 3? any specific issue

Reply:
No specific reason, just wanted to reduce the number, 3 seems just enough but not a strong opinion